specification/transport_layer/can.tex

```diff
@@ -33,26 +33,30 @@ \subsection{CAN ID structure}
             \multicolumn{4}{c|}{Service, not message} &
             \multicolumn{5}{c|}{Anonymous message} &
             \multicolumn{12}{c|}{\multirow{2}{*}{Subject-ID}} &
-            \multicolumn{8}{c|}{Protocol version}
-            \\\cline{2-4} \cline{7-10} \cline{23-29}
+            \multicolumn{1}{c|}{\multirow{2}{*}{R}} &
```

Comment:
so how does this let us know the difference between a v0 and a v1 frame?

Reply:
v0/v1 can be differentiated through the toggle bit, as I wrote here (search for the phrase "short answer"): https://forum.uavcan.org/t/uavcan-v1-0-and-ardupilot/671/9?u=pavel.kirienko.

Reply:
Oh. neat! (Not sure how I missed that). Okay, we need two add to non-normative notes to this PR so others don't miss this important part of the design:
1. In section 4.2.3.2 we need to call out that this is inverted from v0 and can be used now for v0<>v1 determination but in the future can be switched to allow v1<>v2 determination (i.e. only two major versions of UAVCAN can coexist on a single network).
2. Somewhere in this section, when talking about reserved bits, we should mention that the reserved bits cannot be used for protocol level detection and either reiterate the toggle bit requirements or link to that section.

Reply:
Covered this in the new commit just now.